### PR TITLE
Updated readme with a tutorial on how to add an op

### PR DESCRIPTION
**Pull request by remi-or**
and a small fix that may break build (but not yet!)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,9 @@ dist/
 wheels/
 *.egg-info
 
+# CMake generated files
+*.so
```
Comment (ivarflakstad) [inline .gitignore:10]: 👌

File: setup.py [hunk 2]
```diff
@@ -292,6 +292,7 @@ def get_path(*filepath) -> str:
 # Don't ask why I don't know
 root = os.path.dirname(__file__)
 for dir_name in os.listdir(os.path.join(root, "csrc", "op_src")):
+    continue
```
Comment (ivarflakstad) [inline setup.py:297]: 🤔

**Review by ivarflakstad — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (ivarflakstad) [reply setup.py:297]: Maybe add a todo or note as to why it is now `continue` instead of the check you had earlier?

Comment (ivarflakstad) [reply setup.py:297]: Feel free to merge regardless though 👍

Comment (remi-or) [reply setup.py:297]: Right, on it

**Pull request merged**